AzureMonitorAgent/ama_tst/modules/main.py

```diff
@@ -116,12 +116,17 @@ def run_troubleshooter():
     if (not check_sudo()):
         return
     
-    print(sys.argv)
+    # run all checks from command line
     if len(sys.argv) > 1 and sys.argv[1] == '-A':
         success = check_all(False)
         print_results(success)
         print_next_steps()
         return
+    
+    # run log collector from command line
+    if len(sys.argv) > 1 and sys.argv[1] == '-L':
```

Comment:
JFYI, can be in a future PR but I'd recommend moving to use [argparse](https://docs.python.org/3/library/argparse.html). It makes parsing cmd line arguments much cleaner.

Reply:
I will include this change in the next PR, thanks for the advice!